output/bremen/routes.txt

```diff
@@ -18,8 +18,8 @@ route_id,agency_id,route_short_name,route_long_name,route_desc,route_type,route_
 59652,67107911,52,Kattenturm-Mitte-to-Huchting,,3,https://www.openstreetmap.org/relation/59652,,
 61332,67107911,37,Sebaldsbrück-to-Bf Mahndorf,,3,https://www.openstreetmap.org/relation/61332,,
 61333,67107911,34,Horn-to-Sebaldsbrück,,3,https://www.openstreetmap.org/relation/61333,,
-63040,94911646,750,Bus 750 (Bremen-Dreye-Thedinghausen-Morsum),,3,https://www.openstreetmap.org/relation/63040,,
-70673,-1,,De_DELBUS,,3,https://www.openstreetmap.org/relation/70673,,
+63040,94911646,750,OSM Route No. 63040,,3,https://www.openstreetmap.org/relation/63040,,
+70673,-1,,OSM Route No. 70673,,3,https://www.openstreetmap.org/relation/70673,,
```

Comment:
@mehdisadeghi this seems to be happening for several routes. Why? the names from before seemed fine to me

Reply:
The previous name looks fine to me too! Let me see what has happened.

Reply:
@carocad [the relation page](https://www.openstreetmap.org/relation/63040) shows there is no  `from` and `to` tags to construct the previous form. It could be that I have used a different file.  Perhaps we have to add the file metadata to the generated feed.

Reply:
I found the problem. It will make us slower, I have to read an extra tag called `name` for each relation.